tsconfig.base.json

```diff
@@ -34,6 +34,7 @@
       "@nmit-coursition/api/v1": ["libs/api/v1/src/index.ts"],
       "@nmit-coursition/design-system": ["libs/ui/design-system/src/index.ts"],
       "@nmit-coursition/email": ["libs/email/src/index.ts"],
+      "@nmit-coursition/email/components": ["libs/email/src/components/index.ts"],
```

Comment:
**blocker:** No, please avoid this, only move exports to one above (so that we export only once from `@nmit-coursition/email`. :)

Reply:
I'm not sure how we can import the components from email/components without doing that 🤔

Reply:
You will just export it from `libs/email/src/index.ts`. There should be only one barrel file per package if there's not valid reason and this is not one.